src/rust/lib/system/web/src/event/listener.rs

```diff
@@ -0,0 +1,100 @@
+use crate::prelude::*;
+
+use crate::closure::storage::ClosureFn;
+use crate::closure::storage::OptionalFmMutClosure;
+
+
+
+// ============
+// === Slot ===
+// ============
+
+/// A single event listener slot.
+///
+/// Stores a closure that can be registered as an event listener.
+///
+/// Slot will register the closure as event listener whenever both closure and target are provided.
```

Comment:
So, is "Slot" a "Callback" ?

Reply:
It owns callback and manages its connection with the JS event target.

Documentation shall be clarified, as discussed.